CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### Changed
+
+* Replace `@storybook/addon-knobs` with `@storybook/addon-controls`, `@storybook/addon-toolbars`.
+
```

Comment:
Which function of `addon-toolbars` replaced which one from `addon-knob`?

```suggestion
## unreleased

* Replaced `@storybook/addon-knobs` with `@storybook/addon-controls` and `@storybook/addon-toolbars`.

```

Reply:
The addon-toolbars are for the global knobs, addon-controls are for the local knobs.
The addon-controls don't support globally, so this is why we need to use addon-toolbars.